apps/mark/frontend/src/pages/print_page.tsx

```diff
@@ -18,6 +19,22 @@ export function PrintPage(): JSX.Element {
   assert(typeof ballotStyleId === 'string', 'ballotStyleId is not defined');
   assert(typeof precinctId === 'string', 'precinctId is not defined');
 
+  const printerTimer = useRef(0);
+
+  function onPrintStarted() {
```

Comment:
because it's being passed as a prop below, should this be an `onCallback` in principle?

Reply:
_(I'm interpreting your question as whether this should use `useCallback`–if you meant something else, please correct me)_

I've gone back and forth on this. I started out just using functions normally, then I used to try to use `useCallback` on everything that could potentially cause a re-render if it changed, but after reading things like [this](https://kentcdodds.com/blog/usememo-and-usecallback) I've decided to mostly avoid `useCallback` unless there's a specific reason to use it.

Reply:
correct interpretation of my question. had to ask it as I think using `useCallback` was a piece of feedback you gave on an early PR of mine 😉 . makes sense to not use it.